spec/lib/evss/disability_compensation_form/form_8940_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'evss/disability_compensation_form/form8940'
```

Comment:
not needed because we are using autoloading

Reply:
Thanks, I'll remove it